package.js

```diff
@@ -1,33 +1,36 @@
 Package.describe({
+  name: 'fourseven:scss',
+  version: '5.0.0-alpha.1',
   summary: 'Style with attitude. Sass and SCSS support for Meteor.js.',
-  version: '4.17.0-rc.0',
   git: 'https://github.com/Meteor-Community-Packages/meteor-scss.git',
-  name: 'fourseven:scss',
+  documentation: 'README.md'
 });
 
 Package.registerBuildPlugin({
   name: 'compileScssBatch',
   use: [
-    'caching-compiler@2.0.0',
-    'ecmascript@0.16.9'
+    'caching-compiler@1.2.2 || 2.0.0',
+    'ecmascript@0.16.7 || 0.16.10',
   ],
   sources: ['plugin/compile-scss.js'],
   npmDependencies: {
-    "node-sass": '9.0.0',
-    "@babel/runtime": "7.24.5"
+    '@babel/runtime': '7.26.0',
```

Comment:
Is it necessary to pin this version?

Reply:
You're absolutely right! However, without this, the TinyTest GUI crashes. 

It's only needed for that, and I agree that it's quite wasteful. The entire test setup should be modernized as well.

Reply:
You can define this as test only by calling nom depends within onUse and onTest separately as long as there is no npm.depends outside of them